design_notes/grouper_objects.md

```diff
@@ -136,18 +152,22 @@ weighted_sum = xr.dot(ds, weights)
 
 assert_identical(gb_sum, weighted_sum)
 ```
+
 For example, the boolean weights for `group=np.array(['a', 'b', 'c', 'a', 'a'])` should be
+
 ```
 [[1, 0, 0, 1, 1],
  [0, 1, 0, 0, 0],
  [0, 0, 1, 0, 0]]
 ```
-This is the boolean "summarization matrix" referred to in the classic Iverson (1980, Section 4.3)[^2] and "nub sieve" in [various APLs](https://aplwiki.com/wiki/Nub_Sieve).
 
-> [!NOTE]
+This is the boolean "summarization matrix" referred to in the classic Iverson (1980, Section 4.3)\[^2\] and "nub sieve" in [various APLs](https://aplwiki.com/wiki/Nub_Sieve).
+
+> \[!NOTE\]
```

Comment:
Same here — are we sure these should be escaped? Looks like a directive (though not sure what...)